operation.go

```diff
@@ -0,0 +1,7 @@
+package getter
+
+// Operation is the result of a get
+type Operation struct {
```

Comment:
This feels more like a `GetResult` or `GetOutput`, as `Get` returns it only after the Get operation completes.  I would have expected `Operation` to represent the underlying operation to download or upload files, and expose more metadata with some asynchorinisty/cancellations/progress hooks.  Some familiar APIs that use Operations are (Google API [1], AWS SDK [2]).

[1] https://godoc.org/google.golang.org/api/compute/v1#Operation
[2] https://docs.aws.amazon.com/sdk-for-go/api/aws/request/#Operation

Reply:
Thanks for reviewing 🙏. Oh that is a super valid statement. Changing this to `GetResult` !

Reply:
I also removed the operation file and put `GetResult` in client.go.